pkgs/development/libraries/olm/default.nix

```
@@ -34,11 +34,21 @@ stdenv.mkDerivation rec {
       disclaims that its implementations are not cryptographically secure
       and should not be used when cryptographic security is required.
 
-      It is not known that the issues can be exploited over the network in
-      practical conditions. Upstream has stated that the library should
-      not be used going forwards, and there are no plans to move to a
-      another cryptography implementation or otherwise further maintain
-      the library at all.
+      Upstream does not believe this to be a high-severity issue for Matrix
+      because an attack would require sending very large volumes of messages
+      with very accurate timings targeting the same key material. In practice,
+      Matrix implementations have short-lived AES key material (due to
+      ratcheting), very noisy timings (due to traffic topology being client <->
+      server <-> server <-> client, with persistence steps at each hop), as
+      well as traffic rate-limiting. As a result, upstream does not believe
+      that an attack over the network is feasible. It is possible that a local
+      attack might work – but if the attacker has local access to your device,
+      upstream considers you to already be compromised.
+
+      Upstream is encouraging clients to migrate away from libolm to vodozemac,
+      and have stated that libolm should not be used going forwards and that
+      there are no plans to move to a another cryptography implementation or
+      otherwise further maintain libolm at all.
```

I think that if we make a change here at all it should just be:

```suggestion
      It is not known that the issues can be exploited over the network in
      practical conditions. Upstream does not believe such an attack is
      feasible, but has stated that the library should not be used going
      forward, and there are no plans to move to a another cryptography
      implementation or otherwise further maintain the library at all.
```

and letting the links below stand for themselves rather than getting too into the weeds about one party’s statement on the matter.